example/otel-collector/README.md

```diff
@@ -13,165 +13,17 @@ App + SDK ---> OpenTelemetry Collector ---|
 
 # Prerequisites
 
-You will need access to a Kubernetes cluster for this demo. We use a local
-instance of [microk8s](https://microk8s.io/), but please feel free to pick
-your favorite. If you do decide to use microk8s, please ensure that dns
-and storage addons are enabled
+You will need [Docker Compose V2](https://docs.docker.com/compose/) installed for this demo.
 
-```bash
-microk8s enable dns storage
-```
-
-For simplicity, the demo application is not part of the k8s cluster, and will
-access the OpenTelemetry Collector through a NodePort on the cluster. Note that
-the NodePort opened by this demo is not secured.
-
-Ideally you'd want to either have your application running as part of the
-kubernetes cluster, or use a secured connection (NodePort/LoadBalancer with TLS
-or an ingress extension).
-
-If not using microk8s, ensure that cert-manager is installed by following [the
-instructions here](https://cert-manager.io/docs/installation/).
-
-# Deploying to Kubernetes
-
-All the necessary Kubernetes deployment files are available in this demo, in the
-[k8s](./k8s) folder. For your convenience, we assembled a [makefile](./Makefile)
-with deployment commands (see below). For those with subtly different systems,
-you are, of course, welcome to poke inside the Makefile and run the commands
-manually. If you use microk8s and alias `microk8s kubectl` to `kubectl`, the
-Makefile will not recognize the alias, and so the commands will have to be run
-manually.
-
-## Setting up the Prometheus operator
+# Deploying to docker compose
 
-If you're using microk8s like us, simply do
+This command will bring up the OpenTelemetry Collector, Jaeger, and Prometheus, and
+expose the necessary ports for you to view the data.
 
 ```bash
-microk8s enable prometheus
+docker compose up -d
```

Comment:
Should we also call out that users can cleanup using `docker compose down` (or maybe even `docker compose down --remove-orphans`)?

Reply:
Right, I forget this part.